### PR TITLE
Add more test cases

### PR DESCRIPTION
**Pull request by Benson15912**
Add more testcases

### CODE REVIEW

**Review by tayziyi — APPROVED** [review]
lgtm

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 1]
```diff
@@ -89,6 +91,42 @@ public void toStringMethod() {
...
+
+    @Test
+    public void equalTest() {
+        assertTrue(addressBook.equals(addressBook));
+        assertFalse(addressBook.equals("hello"));
+
+        AddressBook other = new AddressBook();
+        addressBook.addPerson(ALICE);
+        other.addPerson(ALICE);
+        assertTrue(addressBook.equals(other));
```
Comment (YowSiaoKang) [inline src/test/java/seedu/address/model/AddressBookTest.java:126]: Can include the case where another person is added and test for false.

**Review by YowSiaoKang — APPROVED** [review]
LGTM

**Review by SimKianSeng — COMMENTED** [review]
Test cases appear robust

**Review by SimKianSeng — APPROVED** [review]
LGTM

**Pull request merged**